### PR TITLE
cleanvm: Support openSUSE:Leap:15.0

### PR DESCRIPTION
**Pull request by toabctl**

### CODE REVIEW

File: jenkins/ci.opensuse.org/openstack-cleanvm.yaml [hunk 1]
```diff
@@ -38,8 +39,8 @@
            ( openstack_project=="Newton" && (image=="SLE_12_SP2" ))
         || ( openstack_project=="Ocata" && (image=="SLE_12_SP3" || image=="openSUSE-Leap-42.3"))
         || ( openstack_project=="Pike" && (image=="SLE_12_SP3" || image=="openSUSE-Leap-42.3"))
-        || ( openstack_project=="Queens" && (image=="SLE_12_SP3" || image=="openSUSE-Leap-42.3"))
-        || ( openstack_project=="Master" && (image=="SLE_12_SP3" || image=="openSUSE-Leap-42.3"))
+        || ( openstack_project=="Queens" && (image=="SLE_12_SP3" || image=="openSUSE-Leap-42.3" || image=="openSUSE-Leap-15.0"))
```
Comment (dirkmueller) [inline jenkins/ci.opensuse.org/openstack-cleanvm.yaml]: I'd prefer if we keep non-master at one sle and max one opensuse distro to test

Comment (toabctl) [reply jenkins/ci.opensuse.org/openstack-cleanvm.yaml]: @dirkmueller then we do not catch Queens+Leap15.0 problems (while we still can fix things in Leap15.0). And only having Leap15.0 (which is not released yet) is imo not an option.

**Comment by toabctl** [comment]
> Repochecker update missing?

Done

File: jenkins/ci.opensuse.org/openstack-repocheck.yaml [hunk 2]
```diff
@@ -39,8 +40,8 @@
     execution-strategy:
       combination-filter: |
        !(["Cloud:OpenStack:Master"].contains(project) && ["SLE_12_SP2"].contains(repository) ||
```
Comment (dirkmueller) [inline jenkins/ci.opensuse.org/openstack-repocheck.yaml]: Leap15 is now also enabled for master? Otherwise it needs to be excluded here

Comment (toabctl) [reply jenkins/ci.opensuse.org/openstack-repocheck.yaml]: I would like to have it enabled for now

**Comment by bmwiedemann** [comment]
I think, I'll propose a follow-up cleanup where we use $VERSION for repo setup of all Leap versions

**Pull request merged**